spec/database_connection_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'database_connection'
+
+describe DatabaseConnection do
```

Comment:
Really like the user of setting up a database in this manner. Makes codebase a lot cleaner.